### PR TITLE
[#63] Adding documentation files

### PR DESCRIPTION
**Pull request by JKal24**
Added markdown files to indicate how changes should be made to the deployed site, how to handle currently featured and all featured data & how to format the data.
Also added some markdown files explaining the forms on the site, their layout and what they should be used for.
Created a template Yaml file that can be filled out and placed in the data folder as a currently featured Yaml file.

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## How to deploy changes to Netlify
+
+### Using Github
+
+The web application is being deployed through Netlify and is linked directly to the Github Repository.
+Changes to data files should be pushed to the Github repository and a new build will be automatically deployed through Netlify.
+
+### Updating
+
```
Comment (dhari001) [inline docs/deploy.md:9]: for some people they'll be updating for the first time and they're going to need an indication that they should clone the repo first so just add a brief instruction about that

File: docs/feature.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      twitter: "https://twitter.com/home"
+      internet: "https://www.google.ca/"
+```
+
+## Currently Featured Files
+
+These files are stored in the `humans/humans-proj/src` folder.
+Every file should have a unique ID associated with it which will represent the file name.
+The ID will be comprised of the date, the person's first name and their last name. 
+The ID is represented as `Date_FirstNameLastName` in the Yaml file and the file name will be `DateFirstNameLastName.yaml`
```
Comment (dhari001) [inline docs/feature.md:51]: what is the format of the date? is it just DD/MM/YYYY or is it in words so "April2nd2022". Specify date format.

Comment (dhari001) [reply docs/feature.md:51]: nvm see the format below

**Review by Fatimbit — CHANGES_REQUESTED** [review]
This is really good! It's great how thorough you are with the instructions to even explain stuff like using cmd and cd path commands and it's very clear and easy to follow. I also thought that having a separate template file was a good idea. 
There are 2 things that should be taken care of before we merge this, which I'm listing below, but this is overall very good!
1. I agree with what Dharitri said about adding a brief instruction about cloning the repo first for people updating for the first time
2. in deploy.md on line 23 it talks about cloning with `git clone https://github.com/Software-For-Love`, but shouldn't that be 'https://github.com/Software-For-Love/humans.git' if they're just trying to clone into our repo for humans

**Review by Fatimbit — APPROVED** [review]
Nice work, thank you!

**Pull request merged**